ec2-spot-terraform/main.tf

```diff
@@ -0,0 +1,96 @@
+variable "region" {
```

Comment:
Add author and comments of what this does

Reply:
Rename perhaps the file to explain the difference between the different ASG configuration, in this case you are using weights, make it easy for users and us in the future to identify what anything does and how to navigate repositories of information